cmd/manager/main.go

```diff
@@ -147,3 +162,23 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// HasKappApis checks whether the cluster has Kapp APIs installed in the cluster.
+// This does not guarantee that the controller is present to reconcile the App CRs.
+func HasKappApis(config *rest.Config) (bool, error) {
```

Comment:
Nit: Doesn't really matter in `main` package since nothing can import a main package, but I'd suggest making this unexported to avoid confusion.

```suggestion
func hasKappApis(config *rest.Config) (bool, error) {
```